docs/root/modules/ontology/schema.md

```diff
@@ -163,3 +164,48 @@ API keys are used across different cloud providers and SaaS platforms for authen
     ```
     (:User)-[:OWNS]->(:APIKey)
     ```
+
+
+
+### ComputeInstance
+
+```{note}
+ComputeInstance is a semantic label.
+```
+
+A compute instance represents a virtual machine or server instance running in a cloud environment.
+It generalizes concepts like EC2 Instances, DigitalOcean Droplets, and Scaleway Instances.
+
+| Field | Description |
+|-------|-------------|
+| _ont_id | The unique identifier for the instance. |
+| _ont_name | The name of the instance. |
+| _ont_region | The region or zone where the instance is located. |
+| _ont_public_ip_address | The public IP address of the instance. |
+| _ont_private_ip_address | The private IP address of the instance. |
+| _ont_state | The current state of the instance (e.g., running, stopped). |
```

Comment:
We use the word "state" here but "status" for containers. Both seem to have similar values (running, suspended, terminated). I think it would be better to align on one word. Also, it would be nice if we standardized the states across compute instances and containers across all clouds. This can be done in a later PR though